projects/humble/index.js

```diff
@@ -0,0 +1,23 @@
+const axios = require('axios');
+const retry = require('../helper/retry');
+const { toUSDTBalances } = require('../helper/balances');
+
+async function tvl(){
+    const response = (
+        await retry(
+            async () => await axios.get(
```

Comment:
Can this be calculated using on-chain calls instead? We've got a guide on how to use our SDK here: https://docs.llama.fi/list-your-project/submit-a-project

Reply:
From what I understand in the docs the SDK only supports EVM. Humble Swap runs on Algorand at the moment so we went with a fetch adapter.

Reply:
okay, my bad, you are 100% right

Reply:
in these cases it's nice if tvl can be sourced from calls to an external party (eg: calls to a block explorer's api) since that makes it easy to find out how it's being calculated.

just an api that returns a single number that is usd tvl makes it impossible to know how it's calculated

Reply:
also, we have algosdk in our dependencies and that can be used to calculate TVL, actually thats what a lot of algorand adapters do. This is also much better.

Actually we should probably change our docs to reflect that (always give higher preference to sdks or third party methods rather than simple fetch adapters where projects report their own tvls), @waynebruce0x @g1nt0ki